rpa-recorder/6.2/modules/ROOT/pages/getting-started.adoc

```diff
@@ -0,0 +1,62 @@
+= Installing and Starting RPA Recorder
+
+You can install RPA Recorder by manually downloading and running the installer or by starting the recorder from RPA Manager, which provides a link to download and install RPA Recorder.
+
+== Install RPA Recorder
+
+Download the latest RPA Recorder installer from the *Home* page of RPA Manager.
+
+Optionally, you can install RPA Recorder from the message prompt that shows in RPA Manager when you start the recording function.
+
+image:download-rpa-recorder-message.png[Download RPA Recorder message]
+
+[NOTE]
+Enable pop-ups from RPA Manager before downloading RPA Recorder.
+
+== Start RPA Recorder
+
+You can start RPA Recorder as a _standalone_ application or from _RPA Manager_.
+
+* *Standalone*
++
+Start RPA Recorder by opening the application from the *Start* menu or by opening the application's executable file. When you start RPA Recorder as a standalone application:
++
+** RPA Recorder does not request RPA Manager credentials to start recordings.
+** You cannot upload recordings to RPA Manager, you can only save them locally.
+* *RPA Manager*
++
+To start RPA Recorder, click the *Record BPMN* icon (image:record-bpmn-icon.png[Record BPMN icon, 2%, 2%]) in RPA Manager. When you start RPA Recorder from RPA Manager:
++
+** RPA Recorder requires RPA Manager credentials to log in and start recordings.
+** You can upload recordings to RPA Manager.
+
+=== Log in to RPA Recorder
+
+After starting RPA recorder via Record BPMN (image:record-bpmn-icon.png[Record BPMN icon, 2%, 2%]) in RPA Manager:
+
+. Open RPA recorder.
+. Provide your login credentials for user authentication.
+
+image:rpa-logout-screen.png[RPA Logout Screen]
+
+To log out of RPA Recorder, click the *Logout* icon (image:logout-icon.png[Logout icon, 2%, 2%]).
+
+== Considerations for Process Recording
+
+Consider the following scenarios when recording processes with RPA Recorder to ensure the captured events are as accurate and robust as possible. 
+
+* *Recording applications started as an administrator*
+** To capture a process where you need to run an application as an administrator, you must run RPA Recorder as an administrator. Otherwise, RPA Recorder cannot capture the events from the application.
+** Because you cannot start RPA Recorder as an administrator from RPA Manager, this process works only when you start RPA Recorder in standalone mode by right-clicking the executable file or the shortcut and selecting *Run as Administrator*. 
+
+* *Record processes using a single screen*
+** If your setup uses multiple monitors, use only the screen where RPA Recorder shows the *Recording Toolbar* to capture the entire process. 
+** To prevent issues during the recording, avoid opening applications on a different screen or moving windows between screens. 
+
+* *Do not change system settings during a recording*
```

Comment:
suggestion: maybe add explicitely the most evil thing: "especially the display settings such as screen resolution or scaling"